### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by jianyexi**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## What's New (02/18/2020)
+
+### New validation rules
+- OperationIdRequired rule - each operation must has a unique operationId.
+
+### Changed Rule
+- Changed the linter rule BodyTopLevelProperties -- support `systemData` top level property
+- Changed the linter rule PatchBodyParametersSchema -- allow required property which be marked as descrimintor
```
Comment (lirenhe) [inline changelog.md]: Could you work with our PM with the change log? I saw a couple of grammar issue in the above sentence.

File: changelog.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## What's New (02/18/2020)
+
+### New validation rules
+- OperationIdRequired rule - each operation must has a unique operationId.
```
Comment (josefree) [inline changelog.md]: must have

File: changelog.md [hunk 3]
```diff
@@ -1,5 +1,14 @@
...
+
+### Changed Rule
+- Changed the linter rule BodyTopLevelProperties -- support `systemData` top level property
```
Comment (josefree) [inline changelog.md]: support 'systemData' as top level property

File: changelog.md (hunk 1 above)
Comment (josefree) [inline changelog.md]: all required property to be discriminator

Updates are proposed above.  Let me know if anything else need my comments.

File: changelog.md (hunk 2 above)
Comment (jianyexi) [reply changelog.md]: thanks, could you please review this again, I have modified it

**Pull request merged**